packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -167,7 +170,12 @@ class FileCollection extends DocumentCollection {
     if (executable === undefined) {
       executable = false
     }
-    const path = uri`/files/${dirId}?Name=${name}&Type=file&Executable=${executable}`
+    let metadataId = ''
```

Comment:
Should this be `null` by default, so that a file without `metadata` doesn't end up with a `metadata` field?

Reply:
If `metadataID` is empty, it will be ignored by the stack: https://github.com/cozy/cozy-stack/blob/master/web/files/files.go#L1189 , so there won't be any `metadata` field.
However a null value will be interpreted as 'null' string by the stack, causing an error.